services/harmony/app/util/parameter-parsers.ts

```diff
@@ -220,3 +220,79 @@ export function handleAveragingType(
     operation.average = value;
   }
 }
+
+/**
+ * Handle the forceAsync parameter in a Harmony query, adding it to the DataOperation
+ * if necessary.
+ *
+ * @param operation - the DataOperation for the request
+ * @param query - the query for the request
+ */
+export function handleForceAsync(
+  operation: DataOperation,
+  query: Record<string, string | boolean>): void {
+  if (query.forceasync !== undefined) {
```

Comment:
I think this code should be pulled out into a separate function like `queryValueToBoolean` that can be used here and in the other two cases. That would make it easy to unit test as well.

Reply:
The three boolean field values are parsed differently as to if undefined is allowed and when will the default value apply. I thought about it hard and ended up with the current code.

Reply:
As I was refactoring the code, I realized that your suggestion is a great idea.